playground/backend-integration/vite.config.js

```diff
@@ -28,6 +28,9 @@ function BackendIntegrationExample() {
           strictPort: true,
           origin: 'http://localhost:5009',
         },
+        preview: {
+          port: 5009,
```

Comment:
`server.strictPort` is inherited to `preview.strictPort` and therefore the playground tried to listen to port 4173 even if one of the other playground is listening to it.
Specified the port number to avoid conflict.